extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -273,17 +274,38 @@ const renderInlinePageAttachmentUi = (pageEl, attachmentEl) => {
 
   const openImgAttr2 = attachmentEl.getAttribute('cta-image-2');
   if (openImgAttr2) {
-    chipEl.prepend(makeImgElWithBG(openImgAttr2));
+    const src = maybeMakeProxyUrl(openImgAttr2, pageEl);
+    chipEl.prepend(makeImgElWithBG(src));
   }
 
   const openImgAttr = attachmentEl.getAttribute('cta-image');
   if (openImgAttr) {
-    chipEl.prepend(makeImgElWithBG(openImgAttr));
+    const src = maybeMakeProxyUrl(openImgAttr, pageEl);
+    chipEl.prepend(makeImgElWithBG(src));
   }
 
   return openAttachmentEl;
 };
 
+/**
+ * Makes a proxy URL if document is served from a proxy origin. No-op otherwise.
+ * @param {string} url
+ * @param {!Element} pageEl
+ * @return {string}
+ */
+const maybeMakeProxyUrl = (url, pageEl) => {
+  const urlService = Services.urlForDoc(pageEl);
+  const loc = pageEl.getAmpDoc().win.location;
+  if (!urlService.isProxyOrigin(loc.origin) || urlService.isProxyOrigin(url)) {
+    return url;
+  }
+  url = urlService.resolveRelativeUrl(
```

Comment:
I wonder if we want to make a new variable here called something like `resolvedRelativeUrl` to make this more readable rather than overriding the argument.

Reply:
Done